src/tests/ftest/util/daos_utils.py

```diff
@@ -36,7 +38,35 @@ class DaosCommand(CommandWithSubCommand):
         # as below:
         # 03/31-21:32:24.53 wolf-3 2f69b198-8478-472e-b6c8-02a451f4de1b
         # UUID is made up of 36 characters of hex and -.
-        "pool_list_cont": r"([0-9a-f-]{36})"
+        "pool_list_cont": r"([0-9a-f-]{36})",
+        # Sample pool query output.
+        # 04/19-18:31:26.90 wolf-3 Pool 3e59b386-fda0-404e-af7e-3ff0a38d1f81,
+        #    ntarget=8, disabled=0
+        # 04/19-18:31:26.90 wolf-3 Pool space info:
+        # 04/19-18:31:26.90 wolf-3 - Target(VOS) count:8
+        # 04/19-18:31:26.90 wolf-3 - SCM:
+        # 04/19-18:31:26.90 wolf-3   Total size: 1000000000
+        # 04/19-18:31:26.90 wolf-3   Free: 999997440, min:124999680,
+        #     max:124999680, mean:124999680
+        # 04/19-18:31:26.90 wolf-3 - NVMe:
+        # 04/19-18:31:26.90 wolf-3   Total size: 0
+        # 04/19-18:31:26.90 wolf-3   Free: 0, min:0, max:0, mean:0
+        # 04/19-18:31:26.90 wolf-3 Rebuild idle, 0 objs, 0 recs
+        "pool_query": r"(?:Pool\s*([A-Za-z0-9-]+),\s*ntarget=([0-9])," +
+            "\s*disabled=([0-9])|Target\(VOS\) count:\s*([0-9])|" +
+            "(?:SCM:\s+.*|NVMe:\s+.*)Total\s+size:\s+([0-9]+)\s+.*" +
+            "Free:\s+([0-9]+),\s+min:([0-9]+),\s+max:([0-9]+),\s+" +
```

Comment:
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.
(pylint-bad-continuation) Wrong continued indentation (add 10 spaces).